### PR TITLE
Add new response fields

### PR DESCRIPTION
**Pull request by darkngel82**
Added the new response fields:
-UVIndex, UVIndexTime
-TemperatureLow, TemperatureLowTime, TemperatureHigh, TemperatureHighTime
-ApparentTemperatureLow, ApparentTemperatureLowTime, ApparentTemperatureHigh, ApparentTemperatureHighTime

EDIT: Also have added the new lib version, and the Alerts new fields ;)

### CODE REVIEW

**Review by kevinzetterstrom — COMMENTED** [review]
Awesome, thank you for this. I've been meaning to do this but have been pretty busy with other work. Can you also increment `libraryVersion` in the gradle file?

**Comment by darkngel82** [comment]
Done, I've increased to 1.2.1.
Thanks to you, it make's me save a lot of time ;) Good job!

File: forecast/src/main/java/android/zetterstrom/com/forecast/models/Alert.java [hunk 1]
```diff
@@ -40,6 +41,14 @@
     @SerializedName(ModelConstants.FIELD_TITLE)
     private String mTitle;
 
+    @Nullable
+    @SerializedName(ModelConstants.FIELD_REGIONS)
+    private List<String> mRegions = null;
```
Comment (kevinzetterstrom) [inline forecast/src/main/java/android/zetterstrom/com/forecast/models/Alert.java]: No need to initialize to null, as it is the default

File: forecast/src/main/java/android/zetterstrom/com/forecast/models/Alert.java [hunk 2]
```diff
@@ -13,14 +13,15 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package android.zetterstrom.com.forecast.models;
+package com.coditramuntana.festinasoprod.forecast.models;
```
Comment (kevinzetterstrom) [inline forecast/src/main/java/android/zetterstrom/com/forecast/models/Alert.java]: Can you change the package name back?

File: forecast/src/main/java/android/zetterstrom/com/forecast/models/ModelConstants.java [hunk 3]
```diff
@@ -13,7 +13,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package android.zetterstrom.com.forecast.models;
+package com.coditramuntana.festinasoprod.forecast.models;
```
Comment (kevinzetterstrom) [inline forecast/src/main/java/android/zetterstrom/com/forecast/models/ModelConstants.java]: Can you change the package name back?

**Review by kevinzetterstrom — COMMENTED** [review]
Nice changes, just a couple of comments.

File: forecast/src/main/java/android/zetterstrom/com/forecast/models/ModelConstants.java (hunk 3 above)
Comment (darkngel82) [reply forecast/src/main/java/android/zetterstrom/com/forecast/models/ModelConstants.java]: Done, sorry!

File: forecast/src/main/java/android/zetterstrom/com/forecast/models/Alert.java (hunk 2 above)
Comment (darkngel82) [reply forecast/src/main/java/android/zetterstrom/com/forecast/models/Alert.java]: Done, sorry!

**Comment by kevinzetterstrom** [comment]
Uploaded to bintray - should be available via jcenter soon! I'll update readme as well

**Comment by batwolf** [comment]
Thank you so much!

**Pull request merged**